curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/use-the-value-attribute-with-radio-buttons-and-checkboxes.md

```diff
@@ -0,0 +1,122 @@
+&#60;label for="outdoor"&#62; <br>&nbsp;&nbsp;&#60;input id="outdoor" value="outdoor" type="radio" name="indoor-outdoor"&#62;Outdoor <br>&#60;/label&#62;
+</blockquote>
+
+Here we have two grouped <code>radio button</code> inputs. When the user submits the form with the "indoor" <code>radio button</code> selected, the form data will include the line <code>indoor-outdoor=indoor</code>. This is the <code>name</code> attribute of <code>indoor-outdoor</code> and the <code>value</code> attribute of <code>indoor</code>.
+
+If you omit the <code>value</code> attribute the submitted form data uses the default value of <code>on</code>. In this scenario, if the user clicked on the "indoor" option and submitted the form, the resulting form data would be <code>indoor-outdoor=on</code>. Which is not useful for identifying which of the two grouped <code>radio button</code>s the user selected. So we need to set the value attribute to something which identifies the choice. In the above example, we used the label text indoor and outdoor as the values for the <code>value</code> attributes.
+</section>
+
+## Instructions
+<section id='instructions'>
+Give each of the <code>radio button</code> and <code>checkbox</code> inputs the <code>value</code> attribute. Use each of the inputs label text, in lowercase, as the value for the <code>value</code> attribute.
+</section>
+
+## Tests
+<section id='tests'>
+
+```yml
+tests:
+  - text: One of your radio buttons should have the <code>value</code> attribute of <code>indoor</code>.
+    testString: assert($('label > input[type="radio"]').filter("[value='indoor']").length > 0, 'One of your radio buttons should have an <code>value</code> attribute with <code>indoor</code>.');
+  - text: One of your radio buttons should have the <code>value</code> attribute of <code>outdoor</code>.
+    testString: assert($('label > input[type="radio"]').filter("[value='outdoor']").length > 0, 'One of your radio buttons should have an <code>value</code> attribute with <code>outdoor</code>.');
+  - text: One of your checkboxs should have the <code>value</code> attribute of <code>loving</code>.
+    testString: assert($('label > input[type="checkbox"]').filter("[value='loving']").length > 0, 'One of your checkboxs should have an <code>value</code> attribute with <code>loving</code>.');
+  - text: One of your checkboxs should have the <code>value</code> attribute of <code>lazy</code>.
+    testString: assert($('label > input[type="checkbox"]').filter("[value='lazy']").length > 0, 'One of your checkboxs should have an <code>value</code> attribute with <code>lazy</code>.');
+  - text: One of your checkboxs should have the <code>value</code> attribute of <code>energetic</code>.
+    testString: assert($('label > input[type="checkbox"]').filter("[value='energetic']").length > 0, 'One of your checkboxs should have an <code>value</code> attribute with <code>energetic</code>.');
```

Comment:
```suggestion
    testString: assert($('label > input[type="checkbox"]').filter("[value='energetic']").length > 0);
```
I would have not touched this if the spelling of `checkboxes` was correct. The assert strings are now taken from the `text` above, so we can completely remove this message